README.md

```diff
@@ -1,2 +1,7 @@
 # v4.loopback.io
-LoopBack 4 Web Site
+![LoopBack 4 logo](http://loopback.io/images/branding/mark/blue/loopback.jpg | width=100)
```

Comment:
When I preview this, I don't get the image. Is this the way it's supposed to be?